indecision-plaza/app/entities/business.ts

```diff
@@ -27,25 +28,25 @@ export class Business {
    *
    */
   public constructor(json) {
-    this.name = json.name;
-    this.phoneNumber = json.display_phone;
-    this.distance = (json.distance / Business.METERS_IN_MILE).toFixed(1);
-    this.imageUrl = json.image_url;
-    this.basicAddress = json.location.address1;
-    this.rating = json.rating;
-    this.price = json.price;
+    this.name = LangUtils.exists(json.name) ? json.name : "Press the button to get results!";
```

Comment:
Use this default to indicate "No business", and have this string propagate through the constructor for `info-card`